Switch/bidi/Switch.js

```diff
@@ -1,19 +1,34 @@
 define([
-	"dcl/dcl"
+	"dcl/dcl",
+	"dpointer/events"
 ], function (dcl) {
-	// module:
-	//		deliteful/Switch/bidi/Switch
 
 	return dcl(null, {
-		_setCheckedLabelAttr: function (value) {
-			value = this.wrapWithUcc(value);
-			this._set("checkedLabel", value);
+		refreshRendering: function (oldVals) {
+			var div;
```

Comment:
as this is starting to be a little bit more than just a couple of lines of code we might want to do them inside a has("bidi") flag to scope it to use-cases where it is needed?